org.freedesktop.dabrain34.squared.json

```diff
@@ -0,0 +1,41 @@
+{
+  "app-id": "org.freedesktop.dabrain34.squared",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "squared",
+  "finish-args": [
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--share=ipc",
+    "--socket=x11",
+    "--persist=.local/share/godot/app_userdata/squared"
+  ],
+  "modules": [
+    {
+      "name": "squared",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://gitlab.freedesktop.org/dabrain34/squared/uploads/2c12efce847d3db39e9e32a8aacb7175/squared-0.1.0.tar.xz",
+          "sha256": "34757a44778f21a2033460edca6d56d5cbb44caa7b51d7436e63100c61e0341f"
```

Comment:
Failed to download sources: module squared: Wrong sha256 checksum for squared-0.1.0.tar.xz, expected "34757a44778f21a2033460edca6d56d5cbb44caa7b51d7436e63100c61e0341f", was "a3dee30e42a9c983df7bfbacb4d43c66754fc4e1e82e875626c1cb57396321b7"